### ISSUE TITLE
[Installation Guide] NixOS && FreeBSD

### ISSUE THREAD

**Issue by wizerer** [psifidotos/applet-window-appmenu#224]
Good day!
Can anyone tell me how to install this great extension for KDE Plasma + Latte in the following Linux distributions:
- **NixOS**
- **Clear Linux**
as well as in **FreeBSD**?

The question is about what additional packages need to be installed for proper dependencies and the application to work.

This extension is so great that I would say it would be very nice if it was implemented in Latte-Dock by default. ))

**Comment by alexander-zolkin** [comment]
Yeah, that would be a good help! I don't want to experiment with Qt installations, for example.
